packages/redux/src/forms/types/state.types.ts

```diff
@@ -0,0 +1,15 @@
+import type { CombinedState } from 'redux';
+import type { Error } from '@farfetch/blackout-client/types';
+import type { FormSchema } from '@farfetch/blackout-client/forms/types';
+
+export type State = CombinedState<{
```

Comment:
This state has several properties that are typed as an object that is indexed with a number key but these properties are indexed with the schemaCode property which is a string. So, I would change the type to string and I would replace `x:` with `schemaCode:`